apps-rendering/src/bodyElement.ts

```diff
@@ -111,13 +112,8 @@ interface AudioAtom {
 	title: string;
 }
 
-interface NewsletterSignUp {
+interface NewsletterSignUp extends Omit<Newsletter, 'theme'> {
```

Comment:
We have `theme` on both the `Newsletter` type and [`Item`](https://github.com/guardian/libs/blob/main/src/format/ArticleFormat.ts#L6). Are we anticipating that they will both have different values? Do you think there's an opportunity to always use the Article's theme?

Reply:
The theme of the Article and the theme of the Newsletter promoted in the article won't necessarily be the same - eg we might promote a 'NEWS' newsletter in an 'OPINION' article . It's the Newsletter's theme that is used to adjust the styling of the sign up component, so we'd need a separate 'theme' field on the `Newsletter`

Reply:
looking back at this bit, the type definition is a bit hard to follow - the idea is that the `NewsletterSignUp` bodyElement should have all same fields as the `Newsletter` API model, except that the `theme` should be an ArticleTheme enum value rather than a string - it would probably be better to keep it as a string to match the API model and convert it to the enum value as needed.